project_generation/content/templates/api/swagger.yaml.tmpl

```diff
@@ -11,6 +11,7 @@ schemes:
   - http
 tags:
   - name: "hello"
+  - name: "Private"
```

Comment:
I think the tags should be consistently cased. Either change this to `private` or the one above to `Hello`